src/Sylius/Bundle/ReviewBundle/Resources/config/validation/Review.xml

```diff
@@ -17,7 +17,7 @@
             <constraint name="NotBlank">
```

Comment:
Shouldn't this constraint be removed now?

Reply:
I don't think so. IMHO, when you write a commented review, you need a title and a comment, when you have a "rating system" by only clicking on stars for example, title is not required.

Reply:
Does it work like that now (title is required if comment is filled)? I guess we'd need some Behat scenario to cover that.

Reply:
No, I just kept existing behaviour on sylius core :)
We can go further but it's more complex. For exemple, you need to separate ratings from commented reviews on frontend. Moreover, the status workflow is different. A single rate doesn't need to be accepted.

Reply:
@pamil I think it's just a first step for sylius. It's just an improvment for sylius review bundle and an easier way to improve sylius review behaviour in a sylius shop.

Reply:
@Zales0123 @pamil @pjedrzejewski @lchrusciel 
Could you think about that, discuss this and tell me what do you think about the best behaviour I could implement.

Currently, with existing Pull request, I added a new validation group (sylius_review) on title to allow using only "sylius" validation group. Another solution could be the idea suggested by Pamil: title is required only if comment is filled. I guess it needs a form listener to do that.

Reply:
Any idea?

Reply:
Sorry @loic425 for the long response. I thing that requiring comment only if title is filled is the best approach, just seems as the most natural solution. /cc @pamil @pjedrzejewski @lchrusciel

Reply:
@Zales0123 Ok, I'm gonna work on that approach.